Landing page

janderson099 opened the pull request.

arctouch-jonathansouza commented:
Hi @janderson099, thanks for sending your PR. I've noticed you used a library to implement the carousel component, however the Code Challenge document  we sent to you says that the candidate must use your own implementation. This requirement is important because it allows us to evaluate your JavaScript skills. Would you mind removing the `slick-carousel` library and implement your own carousel component? Thanks!

janderson099 commented:
Hi @arctouch-jonathansouza, thanks for your feedback.
No problem, I removed <code>slick-carousel</code> and implemented one of my own.
Hope it's all ok now.

If you want, I can show more complex scripts that I did on the past.

Looking forward to hearing from you.

Best regards,
Janderson

arctouch-jonathansouza commented:
Thanks @janderson099. What do you mean with more complex code? Do you feel comfortable with the actual code in this PR? Which one do you prefer?

janderson099 commented:
@arctouch-jonathansouza: Thank you for your reply. I feel confortable doing this kind of code. It was a good challenge to do. What I mean is, I did others codes, like doing a request in a API, getting a Json and parse the datas. I can show if you think it is necessary.

jonathanhds commented:
Got it, thanks again @janderson099. There's no need to send this code, we have all we need :)
